File: iteration2/src/CSE3063F22P1_GRP9/Person.java
package CSE3063F22P1_GRP9;

public abstract class Person {
	private String ID;
	private String firstName;
	private String lastName;
	
	public Person(String ID,String firstName,String lastName) {
		this.ID = ID;
		this.firstName = firstName;
		this.lastName = lastName; 
	}
	
	public abstract void status();
	public String getID() {
		return ID;
	}

	public String getFirstName() {
		return firstName;
	}

	public String getLastName() {
		return lastName;
	}
}

File: iteration3/src/Person.py

from abc import ABC, abstractclassmethod

class Person(ABC):
    def __init__(self, ID, first_name, last_name):
        self._ID = ID
        self._first_name = first_name
        self._last_name = last_name

    @abstractclassmethod   
    def status(self):
        pass
        
    def getID(self):
        return self._ID
    
    def getFirstName(self):
        return self._first_name
    
    def getLastName(self):
        return self._last_name
